starter-code/js/stack.js

```diff
@@ -0,0 +1,28 @@
+$('document').ready(function () {
+
+  var stack1 = new StackDataStructure();
+
+  $("#addBtn").on('click', function () {
+    var input = $("input").val()
+    if (input.length) {
```

Comment:
I'm not craaazy about this condition... it works which is the funny thing about JavaScript but I think its nicer/more explicit to say `input.length > 0`